spec/consensus/light-client/detection.md

```diff
@@ -1,3 +1,68 @@
-# Detection
+# Failure Detection
 
-TODO
+The light client must be able to detect failures in the trust assumptions.
+Ideally, it should never trust a header, which was forged by malicious
```

Comment:
This is true if the trust assumption holds. If it fails, we may trust such a malicious header, but then the key is to detect it, submit evidence, and wait for a human.